### ISSUE TITLE
[forms] Don't include <compat-auto> appearance tests for square-button / push-button / slider-horizontal

### ISSUE THREAD

**Issue by emilio** [web-platform-tests/interop#295]
### Test List

* https://wpt.fyi/results/css/css-ui/appearance-push-button-001.html
* https://wpt.fyi/results/css/css-ui/webkit-appearance-push-button-001.html
* https://wpt.fyi/results//css/css-ui/appearance-slider-horizontal-001.html
* https://wpt.fyi/results//css/css-ui/webkit-appearance-slider-horizontal-001.html
* https://wpt.fyi/results//css/css-ui/appearance-square-button-001.html
* https://wpt.fyi/results//css/css-ui/webkit-appearance-square-button-001.html

### Rationale

These are tests for legacy values that are supposedly needed for compat:

https://drafts.csswg.org/css-ui-4/#typedef-appearance-compat-auto:

>  If any of these values are not needed for web compat, they should be dropped from this list; conversely, any value needed for compat not included in this list should be added.

Firefox doesn't implement these and we haven't seen any compat issue, so I'm proposing getting these removed from the spec in https://github.com/w3c/csswg-drafts/issues/8506.

I don't think we want to implement these just for the sake of passing these tests, so I'd rather get them removed from the interop dashboard (or even from wpt altogether maybe).

**Comment by nt1m** [comment]
I would wait on https://github.com/w3c/csswg-drafts/issues/8506 and remove these tests altogether from WPT if relevant.

**Comment by jgraham** [comment]
Arguably if these properties are little used, and we don't expect/want that to change, we could remove the tests from interop either way. But it also seems fine to wait and see what the CSS-WG decision is.

**Comment by foolip** [comment]
@emilio we discussed this briefly in the interop team meeting today. This is blocked on https://github.com/w3c/csswg-drafts/issues/8506, can you pull some strings to get it on the agenda in a meeting soon?

**Comment by foolip** [comment]
There's a resolution for https://github.com/w3c/csswg-drafts/issues/8506 now. @emilio do you think we can update the tests right away, or wait for spec edit?

**Comment by emilio** [comment]
We can probably just update the tests right away.

**Comment by zcorpan** [comment]
Spec change: https://github.com/w3c/csswg-drafts/pull/8877 (merged)
Tests change: https://github.com/web-platform-tests/wpt/pull/40233 (needs review)

**Comment by foolip** [comment]
Tusen tack @zcorpan!
